cls-getting-started.html

```diff
@@ -170,8 +197,7 @@
 
 <li>
 
-<p class="first"><a class="anchor" name="groups" id="groups"></a>When you log into OMERO for the first time, you will be in a "default" group, which is Private, so only you can see data you place in that group.
-<p>Being a member of a read-only or read-annotate group in OMERO allows you to share data with other members of the group.</p>
+<p class="first"><a class="anchor" name="groups" id="groups"></a>When you log into OMERO for the first time, you will be in a "default" group, My Data, which is Private, so only you can see data you place in that group.  <p>Being a member of a read-only or read-annotate group in OMERO allows you to share data with other members of the group.</p>
```

Comment:
not having a title for this anchor inline is a bit inconsistent, is there a reason for having this as 4. rather than a 'Groups' section?